src/views/About/AboutDetailsItem.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const AboutDetailsItem = ({ item }) => (
+  <div>
+    <h2>{item.title}</h2>
+    <p>{item.contextOne}</p>
+    <p>{item.contextTwo}</p>
+  </div>
+);
+
+AboutDetailsItem.propTypes = {
+  item: PropTypes.instanceOf(Object).isRequired,
```

Comment:
We could improve our type checking here a lot by using `PropTypes.shape` :) `PropTypes.instanceOf(Object)` will only check if we're passing an object to the component. So, for example, we can pass an empty object `{}` as `item` to the component. We will not get the properties we need (`title`, `contextOne`, etc.), but we will not get any warning from PropTypes! Using `PropTypes.shape` will give us much more control over what we pass to the component.

```
AboutDetailsItem.propTypes = {
  item: PropTypes.shape({
    id: PropTypes.number.isRequired,
    title: PropTypes.string.isRequired,
    contextOne: PropTypes.string.isRequired,
    contextTwo: PropTypes.string.isRequired,
  }).isRequired,
};
```